Ruby/syntax_test_ruby.rb

```diff
@@ -1128,6 +1128,8 @@ class MyClass
 exit! 2
 #^^^^ support.function.builtin
 
+ ->
+#^^ meta.function.ruby storage.type.function.ruby keyword.declaration.function.lambda.ruby
```

Comment:
This is a good first step, but perhaps provide a slightly more detailed example also? I'd recommend perhaps some parameters and a simple function body.

Reply:
Got it, I've pushed a better example for the `->`